cobra_test.go

```diff
@@ -513,6 +523,11 @@ func TestSubCmdTakesArgs(t *testing.T) {
 func TestCmdOnlyValidArgs(t *testing.T) {
 	result := noRRSetupTest("echo times one two five")
 
+	if result.Error == nil {
+		t.Errorf("Expected an error")
+		t.FailNow()
```

Comment:
Thanks. I've made that change.  It did seem strange that I had to call two functions, but somehow I missed this.